packages/expo-updates/ios/EXUpdates/Database/EXUpdatesDatabase.m

```diff
@@ -120,6 +120,10 @@ - (void)addNewAssets:(NSArray<EXUpdatesAsset *> *)assets toUpdateWithId:(NSUUID
 
 - (BOOL)addExistingAsset:(EXUpdatesAsset *)asset toUpdateWithId:(NSUUID *)updateId error:(NSError ** _Nullable)error
```

Comment:
Add a docblock explaining that this method returns NO but does not set an error when the asset has no key. Typically iOS APIs return NO/nil and set an error together, and this method is different.